docs/pages/jpscore/jpscore_sources.md

```diff
@@ -11,30 +11,74 @@ last_updated: Dec 31, 2019
 
 ## Parameter of a source
 
-- `id`: id of the source
-- `caption`: caption
-- `frequency`: time in seconds of generation of pedestrians (default = 1).
-- `N_create`: How many agents to create at once (default = 1).
-- `percent`: percent of `N_create` to generate (default = 1).
-- `rate`: rate of generation of agents (in seconds).
-- `time_min`, `time_max`: Time lifespan of the source.
-- `agents_max`: maximal number of agents produced by that source.
+- `id` (int): id of the source
+- `caption` (str): caption (default: "no caption")
+- `frequency` (int): time in seconds of generation of pedestrians (default: 1).
+- `N_create` (int): how many agents to create at once (default: 1).
+- `percent` (float): percent of `N_create` to generate (default: 1.0). A number between 0 and 1.
+
+{%include note.html content="percent should be at least equal to $$\lceil \frac{\text{frequency}}{\text{rate}}\rceil$$ in order to generate `N_create` agents."%}
+
+- `rate` (int): rate of generation of `percent`$$\times$$`N_create` agents. (default: `frequency`)
+- `time_min` (float), `time_max`: Time lifespan of the source.
+- `agents_max` (int): maximal number of agents produced by that source. (default: 10)
 - `group_id`: group id of the agents. This `id` **should match** a predefined group in the section [Agents_distribution](jpscore_inifile.html#agents_distribution).
-- `time`: time of appearance of agent with id `agent_id`. Here `agents_max=1`.
-- `startX`, `startY`: Distribute one agent at a fix position.
-- `x_min`, `x_max`, `y_min`, `y_max`: Bounding box for generation of agents.
-- `greedy` (default `false`): returns a Voronoi vertex randomly with respect to weights proportional to squared distances.
+- `time` (float): time of appearance of agent. Should be used together with `agent_id`. If used then `agents_max=1` and `frequency=1`.
```

Comment:
What is this `agent_id`? Is not explained anywhere. If I understand this correctly, setting `agent_id` will just create an agent with the specific id at time=`time`?